### PR TITLE
update data gen script, gitignore, and eslintrc

### PR DESCRIPTION
**Pull request by naszzz**

### CODE REVIEW

File: db/gen/generate.js [hunk 1]
```diff
@@ -1,62 +1,238 @@
-/* eslint-disable no-undef */
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable camelcase */
+/* eslint-disable no-unused-vars */
```
Comment (augustohalvarez) [inline db/gen/generate.js:3]: Why the no-unused-vars disabled?

File: db/gen/resource.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ module.exports.addictions = [
   'Work',
   'Shopping',
   'Pain',
-  'Exercise'
+  'Exercise',
```
Comment (augustohalvarez) [inline db/gen/resource.js:12]: Why the trailing commas everywhere?

File: db/gen/resource.js [hunk 3]
```diff
@@ -35,12 +35,24 @@ module.exports.cities = [
   'Las Vegas',
   'Baltimore',
   'Atlanta',
-  'Oakland'
+  'Oakland',
 ];
 
-module.exports.lookingFor = [
+module.exports.connectionTypes = [
   'Friend',
```
Comment (dmoygit) [inline db/gen/resource.js:42]: Are these the options we will be showing the users? if so, should we be consistent with the format, i.e.
Looking:
'for a friend'
'for a relationship'
'for a sponsor'
'to sponsor'

OR if you want to use 
Looking for a:
'friend'
'relationship'
'sponsor'
'person to sponsor'

File: db/gen/resource.js [hunk 4]
```diff
@@ -35,12 +35,24 @@ module.exports.cities = [
...
+  'For a Sponsor',
+];
+
+module.exports.genders = [
+  'Male',
+  'Female',
+  'Other',
+];
+
+module.exports.sexualOrientations = [
```
Comment (dmoygit) [inline db/gen/resource.js:54]: Should we add transgender?

**Pull request merged**